### ISSUE TITLE
[Feature Request] Redirect Newsletters

### ISSUE THREAD

**Issue by pabloscloud** [anonaddy/anonaddy#618]
I'd love to redirect newsletters to an address. It could be configurable by adding a rule with a condition 'is newsletter' and action 'redirect to'.

```sieve
anyof (header :matches "List-Unsubscribe" "*",
             header :matches "List-Id" "*",
             header :matches "List-Post" "*",
             header :matches "List-Subscribe" "*",
             header :matches "List-Help" "*")
{
    redirect :copy "mail@example.com";
}
```

Or else a rule to check the body for the word 'Unsubscribe' which worked pretty good in my experience as well.

(My new email provider doesn't support this either, but I need emails to be forwarded to Omnivore)

**Comment by stites** [comment]
Does https://kill-the-newsletter.com / https://github.com/leafac/kill-the-newsletter work for you? If so, I think this can be closed.

**Comment by pabloscloud** [comment]
> Does https://kill-the-newsletter.com / https://github.com/leafac/kill-the-newsletter work for you? If so, I think this issue can be closed.

No, it does not. This would require me to subscribe using the kill-the-newsletter alias LR else all my mails would be sent there
